Codegen/src/Generators/LinearNodeSourceCodeMapper.cs

```diff
@@ -852,40 +799,21 @@ private bool ProcessLinearization(Node node, bool functionBody = false)
                             if (copyToken != null && !MappedCopyTokens.ContainsKey((copyToken)))
                             {//We got one ==>
                                 //Collect all COPY Tokens
-                                bool bEnd = false;
-                                bool bCopySeen = false;
-                                //A Copy of a Commented node must have is lines commented also.
+                                var consumedTokens = node.CodeElement.FirstCopyDirective.ConsumedTokens;
+                                var copyTokens = consumedTokens.SelectedTokensOnSeveralLines.SelectMany(tokenLine => tokenLine).ToList();
+
+                                //A Copy of a Commented node must have its lines commented also.
                                 bool bCopyCommented = node.Comment.HasValue && node.Comment.Value;
-                                int lastLine = -1;
-                                var copyTokens = CopyTokensYielder(node.CodeElement.FirstCopyDirective).Where(t =>
+                                if (bCopyCommented)
                                 {
-                                    if (t == copyToken)
+                                    for (int i = 0; i < consumedTokens.SelectedTokensOnSeveralLines.Length; i++)
```

Comment:
Put outside of loop "consumedTokens.SelectedTokensOnSeveralLines.Length": declare int variable

Reply:
See commit b5bc0397db3656f59efbed729eeea36f93ffbe38